geode-redis/src/main/java/org/apache/geode/redis/internal/ExecutionHandlerContext.java

```diff
@@ -101,13 +100,10 @@ public PubSub getPubSub() {
    * @param pwd Authentication password for each context, can be null
    */
   public ExecutionHandlerContext(Channel ch, Cache cache, RegionProvider regionProvider,
-      GeodeRedisServer server, byte[] pwd, KeyRegistrar keyRegistrar,
-      PubSub pubSub,
-      RedisLockService hashLockService,
-      RedisLockService setLockService) {
+      GeodeRedisServer server, byte[] pwd, KeyRegistrar keyRegistrar, PubSub pubSub,
+      RedisLockService lockService) {
     this.keyRegistrar = keyRegistrar;
-    this.hashLockService = hashLockService;
-    this.setLockService = setLockService;
+    this.lockService = lockService;
     this.pubSub = pubSub;
     if (ch == null || cache == null || regionProvider == null || server == null) {
```

Comment:
`ch` is bad parameter/variable name. Please correct to something a little more descriptive. Like `channel`... makes it easier to read when one gets past 15 lines of code where it was initially defined

Reply:
done. thx